client/clip_client/helper.py

```diff
@@ -52,3 +52,20 @@ def is_latest_version(package: str = None, github_repo: str = None) -> None:
     """
 
     threading.Thread(target=_version_check, args=(package, github_repo)).start()
+
+
+def get_authorization(scheme, credential):
+    if scheme == 'grpc':
+        return (
+            ('Authorization', credential['Authorization'])
+            if 'Authorization' in credential
+            else None
+        )
+    elif scheme == 'http':
+        return (
+            {'Authorization': credential['Authorization']}
+            if 'Authorization' in credential
+            else None
+        )
+    else:
+        return None
```

Comment:
```suggestion
def get_authorization(scheme, credential):
    auth = ()
       
    if crendential and 'Authorization' in crendential:
        auth += (('Authorization': credential['Authorization']),)
    
    if schema == 'http':
        auth = dict(auth)
    return auth
```

Reply:
i think we don't need this and can make it one-liner